feat: add base request handler

slawrence87544 opened the pull request:
body: this code will still be non-functional and will need a
specific request handler.  This handler receives either a dict
or yaml file which describes the type of request (GET, PUT), the
tables to insert or receive data from, and the specific filters.
This will act much like a REST api, but will not require an http
request.  These request handlers could be modified fairly easily
to transform them into actual REST apis.

slawrence87544 committed:
feat: add base request handler

body: this code will still be non-functional and will need a
specific request handler.  This handler receives either a dict
or yaml file which describes the type of request (GET, PUT), the
tables to insert or receive data from, and the specific filters.
This will act much like a REST api, but will not require an http
request.  These request handlers could be modified fairly easily
to transform them into actual REST apis.

frolovsa commented on src/score_db_base.py:
```diff
@@ -0,0 +1,96 @@
+    file_utils.is_valid_readable_file(request_yaml)
+
+    # Submit the score db request
+    handle_request(request_yaml)
```
add return of the response to main?

slawrence87544 replied:
added...

frolovsa reviewed (changes requested):
Looks good. one small request for change. see code comments

slawrence87544 committed:
Merge branch 'develop' into feature/add-base-class-handler-and-function-registry

slawrence87544 committed:
feat: add return method to main

body: the main function within score_db_base.py will now return the response
which was received from the handle_request method.
